Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackAndGenerateCode.cpp

```diff
@@ -144,12 +144,17 @@ static ALWAYS_INLINE CCallHelpers::Address callFrameAddr(CCallHelpers& jit, intp
         return CCallHelpers::Address(GPRInfo::callFrameRegister, offsetFromFP);
     }
 
+    if (isARM()) {
+        // For now, we can just solve this in the macro assembler ...
+        return CCallHelpers::Address(GPRInfo::callFrameRegister, offsetFromFP);
```

Comment:
But you fix it below anyway?

Reply:
I figured this was more of a "when `Addr::isValidForm` makes sense on 32-bit, we don't want more ptr/i32/i64 confusion here" kind of fix but I can take it out if we prefer

note that we don't have `extendedOffsetAddrRegister()` on armv7 though _offhand_ I don't remember why we cannot use the link register for this purpose also, will run off and try to remember